api/src/routes/platform.ts

```diff
@@ -210,47 +50,23 @@ router.patch(
   middleware.inputs([
     ev
       .body('role')
-      .custom((v) =>
-        Object.values(EPlatformRole)
-          .filter(
-            (e: EPlatformRole) =>
-              ![EPlatformRole.Owner, EPlatformRole.Super].includes(e)
-          )
+      .custom((v) => {
+        const platformRoles: EPlatformRole[] = [
+          'guest',
+          'user',
+          'admin',
+          'owner',
+          'super'
+        ]
+        return platformRoles
+          .filter((e: EPlatformRole) => e !== 'owner' && e !== 'super')
```

Comment:
Could we simplify to the following?
```suggestion
        ['guest', 'user', 'admin']
```